google/cloud/bigquery/schema.py

```diff
@@ -291,3 +318,72 @@ def _to_schema_fields(schema):
         field if isinstance(field, SchemaField) else SchemaField.from_api_repr(field)
         for field in schema
     ]
+
+class PolicyTagList(object):
+    """Define Policy Tags for a column.
+
+    Args:
+        names (Union[List[str], None]): list of policy tags to associate with
```

Comment:
(nit) Mypy's [recommendation](https://mypy.readthedocs.io/en/latest/kinds_of_types.html#optional-types-and-the-none-type) for optional arguments is the `Optional[x]` shorthand. Applies to several places.

Reply:
Done